ws/ws.d.ts

```diff
@@ -8,6 +8,7 @@
 declare module "ws" {
     import * as events from 'events';
     import * as http from 'http';
+    import * as http from 'https';
```

Comment:
Hi @mabels,
it should be `import * as https from 'https';`. Please improve it. :smile: